_extras/guide.md

```diff
@@ -52,7 +56,8 @@ Explain that we use Python because:
 *   It's free.
 *   It has a lot of scientific libraries, and more are constantly being added.
 *   It has a large scientific user community.
-*   It's easier for novices to learn than most of the mature alternatives.
+*   It's easier for novices to learn than most of the mature alternatives, 
+especially due to its syntax that reads like a human language.
```

Comment:
What do you think of toning this down a little to say:
```
*   It's easier for novices to learn than most of the mature alternatives, 
partially due to its syntax reading more like a human language.
```

Reply:
partly?